vagrant_support/bmo-reconfigure.j2

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd /vagrant
+rm localconfig data/params
+perl checksetup.pl /home/vagrant/checksetup_answers.txt
+perl checksetup.pl /home/vagrant/checksetup_answers.txt
```

Comment:
Yeeeah. The first time it writes a localconfig file and the second time it creates the database scheme.